Electron/src/assets/i18n/en.json

```diff
@@ -9,8 +12,8 @@
           "INCLUDE_COMMANDER": "Install Commander",
           "INCLUDE_COMMANDER_VSAI": "Vs Vanilla AI",
           "INCLUDE_COMMANDER_OFF": "No Commander",
-          "OPTIMISE": "Use Optimised Scripts",
-          "FORCELANG": "Override Default AI Language"
+          "OPTIMISE": "Reduce System Load",
```

Comment:
We should make an onhover tooltip rather than change the message here.

Reply:
Actually, no one may have read it, but I rely entirely on AI to consult a little bit on HTML adjustments, and the changes may not be as good as those made by professional engineers
like `How to add a standard question mark icon after text, with a mouse move in button prompt`